pkg/types/iface.go

```diff
@@ -30,3 +30,31 @@ func ValidateIfaceNames(ifaceList string) ([]string, error) {
 	}
 	return ifaces, nil
 }
+
+func ValidateAndSeparateFilters(ifaceList string) ([]string, []string, error) {
+	ifaces := strings.Split(ifaceList, ",")
+	var positive, negative []string
+	for _, iface := range ifaces {
+		if err := ValidateIfaceName(iface); err != nil {
+			return nil, nil, err
+		} else {
+			if iface[0] == '!' {
+				negative = append(negative, iface[1:])
+			} else {
+				positive = append(positive, iface)
+			}
+		}
+	}
+	return positive, negative, nil
+}
+
+func ValidateRegExp(regExp string) (*regexp.Regexp, error) {
```

Comment:
Yeah, I thought about not having a validation method, but as empty string is valid regexp and it would probably lead to confusing results for the user, I felt it was more user-friendly to have that empty check,